src/App.js

```diff
@@ -35,7 +35,11 @@ class App extends Component {
   }
 
   findMovie = (match, location) => {
-    const foundMovie = this.state.movies.find(movie => movie.id === parseInt(match.params.id))
+    const foundMovie = this.state.movies.find(movie => {
+      if (match.params.id.length === 6 && movie.id === parseInt(match.params.id)) {
```

Comment:
This was some productive learning and understanding that parseInt removes text when converting a string into a number